django/test/runner.py

```diff
@@ -459,6 +458,10 @@ def build_suite(self, test_labels=None, extra_tests=None, **kwargs):
         for test in extra_tests:
             suite.addTest(test)
 
+        if tags or exclude_tags:
+            tags = tags or set()
```

Comment:
Is there any reason to perform these conversions here rather than inside `filter_tests_by_tags`?